apps/content/schemaTypes/documents/hackathon.ts

```diff
@@ -0,0 +1,235 @@
+      group: 'content',
+    }),
+    defineField({
+      name: 'body',
+      type: 'markdown',
+      description: 'Full hackathon details, rules, and content',
+      validation: (Rule) => Rule.required().error('Body content is required'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'episodes',
+      type: 'array',
+      title: 'Related Episodes',
+      description: 'Episodes associated with this hackathon',
+      of: [
+        defineArrayMember({
+          type: 'reference',
+          to: [{type: 'episode'}],
+          options: {
+            disableNew: true,
+          },
+        }),
+      ],
+      group: 'content',
+    }),
+    defineField({
+      name: 'submissionForm',
+      type: 'url',
+      title: 'Submission Form',
+      description: 'URL to the submission form for this hackathon',
+      validation: (Rule) =>
+        Rule.required()
+          .uri({scheme: ['http', 'https']})
+          .error('Submission form must be a valid URL'),
+      group: 'content',
+    }),
+    defineField({
+      name: 'rewards',
+      type: 'reference',
+      title: 'Rewards',
+      description: 'Rewards collection for this hackathon',
+      to: [{type: 'rewards'}],
+      options: {
+        disableNew: true,
+        filter: '_type == "rewards"',
+      },
+      initialValue: async (_props: any, {getClient}: any) => {
+        const client = getClient({apiVersion: '2024-01-01'})
+        const rewardsId = await client.fetch(
+          '*[_type == "rewards" && slug.current == $slug][0]._id',
+          {
+            slug: 'hackathon-rewards',
+          },
+        )
+        if (rewardsId) {
+          return {_ref: rewardsId}
+        }
+        return {_ref: ''}
+      },
+      group: 'content',
+    }),
+    defineField({
+      name: 'faq',
+      type: 'reference',
+      title: 'FAQ',
+      description: 'FAQ collection for this hackathon',
+      to: [{type: 'faq'}],
+      options: {
+        disableNew: true,
+        filter: '_type == "faq"',
+      },
+      initialValue: async (_props: any, {getClient}: any) => {
+        const client = getClient({apiVersion: '2024-01-01'})
+        const faqId = await client.fetch('*[_type == "faq" && slug.current == $slug][0]._id', {
+          slug: 'hackathon-faq',
+        })
+        if (faqId) {
+          return {_ref: faqId}
+        }
+        return {_ref: ''}
+      },
+      group: 'content',
+    }),
+    defineField({
+      name: 'sponsors',
+      type: 'reference',
+      title: 'Sponsors',
+      description: 'Sponsors for this hackathon',
+      to: [{type: 'sponsor'}],
+      options: {
+        disableNew: true,
+        filter: '_type == "sponsor"',
+      },
+      group: 'content',
+    }),
+    defineField({
+      name: 'rules',
+      type: 'array',
+      title: 'Rules',
+      description: 'Hackathon rules and guidelines',
+      of: [{type: 'ruleItem'}],
```

Comment:
Probably good to follow the same format here by adding default rules and using that to build `initialValue`